### ISSUE TITLE
Replace Databse

### ISSUE THREAD

**Issue by UlisesGascon** [GoblinDBRocks/GoblinDB#15]
To be ensure that databse can be replace ONLY with an object.

**Comment by CodingCarlos** [comment]
If I 've understood well, you want to check every time somebody performs a "set" operation referred to all the DB, that data sent is an object and not an array.

If so, adding another check [on line 246](https://github.com/GoblinDBRocks/GoblinDB/blob/master/goblin.js#L246)  shall be enough. For example, changing:

```javascript
} else if (!point && data && typeof(data) === "object") {
```
to
```javascript
} else if (!point && data && typeof(data) === "object" && !(data instanceof Array)) {
```

shall do the trick. I'm on it in dev. I'm also going to throw a different error when that happens, to let the user know that an Array is not a valid DB format ;)

**Comment by UlisesGascon** [comment]
Great improvement! Thanks!

**Comment by CodingCarlos** [comment]
Fixed in 3af257731be9b851a676e7f398c195ec91e1f73b. Not yet in master.
